arex-storage-web-api/src/main/java/com/arextest/storage/service/mockerhandlers/MockerHandlerFactory.java

```diff
@@ -13,12 +14,13 @@ public class MockerHandlerFactory {
   @Resource
   List<MockerSaveHandler<?>> handlers;
 
-  public <T extends Mocker> MockerSaveHandler<T> getHandler(MockCategoryType type) {
+  public <T extends Mocker> List<MockerSaveHandler<T>> getHandler(MockCategoryType type) {
+    List<MockerSaveHandler<T>> result = new ArrayList<>();
     for (MockerSaveHandler<?> handler : handlers) {
       if (handler.getMockCategoryType().equals(type)) {
-        return (MockerSaveHandler<T>) handler;
+        result.add((MockerSaveHandler<T>) handler);
       }
     }
-    return null;
+    return result;
```

Comment:
Suggest initialize a `Map<Type, List<Handler>>` to avoid compute and create new List